Storage/Storage/Tools/StorageType+Deletions.swift

```diff
@@ -129,4 +129,15 @@ public extension StorageType {
             deleteObject($0)
         }
     }
+
+    // MARK: - System plugins
+
+    /// Deletes all of the stored SystemPlugins for the provided siteID.
+    ///
+    func deleteSystemPlugins(siteID: Int64) {
```

Comment:
I think this function will be insufficient when you get to use it.
We try to only delete objects that are no longer necessary, if you look at the function above, you can see how we only delete stale entities.

⚠️ Do you think we can update this to follow that pattern?

Reply:
Sure! Modified!